exercises/simple-linked-list/linked_list_test.exs

```diff
@@ -0,0 +1,154 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("linked_list.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LinkedListTest do
+  use ExUnit.Case
+
+  test "length/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.length(list) == 0
+  end
+
+  @tag :pending
+  test "empty?/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "length/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(10)
+    assert LinkedList.length(list) == 1
+  end
+
+  @tag :pending
+  test "empty?/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    refute LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "peek/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    assert LinkedList.peek(list) == {:ok, 20}
+  end
+
+  @tag :pending
+  test "peek/1 of list of empty list" do
+    list = LinkedList.new()
+    assert LinkedList.peek(list) == {:error, :empty_list}
+  end
+
+  @tag :pending
+  test "tail/1 of empty list" do
+    list = LinkedList.new()
+    assert {:error, :empty_list} = LinkedList.tail(list)
```

Comment:
Why do you use this assertion style (relying on pattern matching rather than equality) here? It seemed oddly inconsistent to me, and I worry it might confuse others as well. I expected it to look pretty much the same as the assertion on line 43 above.